ncm-altlogrotate/src/main/pan/components/altlogrotate/schema.pan

```diff
@@ -62,7 +62,7 @@ type structure_altlogrotate_logrot = {
     'taboo_replace' ? boolean
     'tabooext' ? string[]
 
-    'frequency' ? string with match(SELF, '^(daily|weekly|monthly)$')
+    'frequency' ? choice('daily','weekly','monthly','yearly')
```

Comment:
Actually this failed panlint: 

ncm-altlogrotate/src/main/pan/components/altlogrotate/schema.pan:65: Commas should be followed by exactly one space
189    'frequency' ? choice('daily','weekly','monthly','yearly')